scripts/upgrade

```diff
@@ -59,6 +59,11 @@ if [ -z "$final_path" ]; then
 	ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 fi
 
+# If lang exist, remove it
+if [ -v "$lang" ]; then
+	ynh_app_setting_delete --app=$app --key=$lang
+fi
```

Comment:
won't work because `lang` is not loaded at the LOAD SETTING step, you should let it like it was